projects/storefrontapp-e2e-cypress/cypress/integration/b2b/accessibility/tabbing-order.e2e-spec.ts

```diff
@@ -131,4 +141,39 @@ describe('Tabbing order for B2B checkout', () => {
       });
     });
   });
+
+  describe('Order Approval', () => {
+    before(() => {
+      cy.window().then((win) => win.sessionStorage.clear());
+      Cypress.env('BASE_SITE', POWERTOOLS_BASESITE);
+    });
+
+    context('Approval List', () => {
+      it('should allow to navigate with tab key', () => {
+        loginB2bApprover();
```

Comment:
Duplication on calling loginB2BApprover can be simplified placing it on a beforeEach() hook. Or even better, just once on before() hook.

Reply:
done, moved to `beforeEach()`.